### PR TITLE
update with naveen try 2

### PR DESCRIPTION
**Pull request by naveenbalachandran**
Ok I am trying again to but me in this

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://naveenbalachandran.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -444,3 +444,11 @@
   image: rahul.jpg
   twitter: 448bb8
   bio: Programmer
+
+  - name: Naveen Balachandran
```
Comment (Yathannsh) [inline _data/students.yml]: There are build errors because of the extra space before 'name'. 
Remove it and this is all set to be merged! :)

**Comment by jigyasa-grover** [comment]
@naveenbalachandran Good going, you have improved a lot from your previous PRs. Just correct this issue and we are ready to merge !

**Comment by jigyasa-grover** [comment]
Great Work @naveenbalachandran ! You finally did it :)
Submit your task for review and you shall be done with it !

**Comment by naveenbalachandran** [comment]
Thanks for helping me. Love you :)

**Pull request merged**